solr/solrj-zookeeper/src/java/org/apache/solr/common/cloud/ZkStateReader.java

```diff
@@ -1149,6 +1150,12 @@ private void loadClusterProperties() {
               ClusterProperties.convertCollectionDefaultsToNestedFormat(properties);
           log.debug("Loaded cluster properties: {}", this.clusterProperties);
 
+          if (!this.clusterProperties.containsKey(URL_SCHEME)) {
```

Comment:
@justinrsweeney solr reads cluster properties from multiple places (URL_SCHEME). Thus adding here that check, when solr reads cluster-props file from zk. Let me know if this looks reasonable